application/config/config-defaults.php

```diff
@@ -95,7 +95,7 @@
     'wav', 'mp3', 'flac', 'aac', 'm4a', 'opus', 'ogg', 'wma', 'mka',
     //videos
     'mp4', 'avi', 'mkv', 'mpeg', 'mpg', 'wmv', 'h264', 'h265', 'mov', 'webm', 'divx', 'xvid',
-]; 
```

Comment:
Please, in the future, do not mix minor clean-up with new features. It makes the PR harder to read. Thank you.

Reply:
Sorry, that was editor defaults.. Will take into consideration next time.

Reply:
Aha! Yes, if possible, disable that feature or make a separate PR with cleanups.

Reply:
Sure,,, is it OK if I do it next time?